source/developer/interactive-messages.rst

```diff
@@ -286,6 +287,18 @@ Context
       }
 
   Then, when the integration receives the request, it can act based on the action id.
+  
+  3. **Authenticating the server**.Sometimes, you might want to include a custome header in requests to be consumed by the integration. In such cases, you could add a Custom Header to your message, and it will be added to the HTTP Header field.
+ 
+    .. code-block:: text
+
+       {
+      "context": {
+        "action_id": "someunguessableactionid"
+        },
+       "custom_header": "some value",
```

Comment:
propose `some value` to `somevalue` which is consistent with the action_id parameter